src/commands/run.ts

```diff
@@ -61,21 +63,20 @@ export class RunCommand extends ApifyCommand<typeof RunCommand> {
 		purge: Flags.boolean({
 			char: 'p',
 			description:
-				'Shortcut that combines the --purge-queue, --purge-dataset and --purge-key-value-store options.',
-			required: false,
-		}),
-		'purge-queue': Flags.boolean({
-			description: 'Deletes the local directory containing the default request queue before the run starts.',
-			required: false,
-		}),
-		'purge-dataset': Flags.boolean({
-			description: 'Deletes the local directory containing the default dataset before the run starts.',
-			required: false,
-		}),
-		'purge-key-value-store': Flags.boolean({
-			description:
-				'Deletes all records from the default key-value store in the local directory before the run starts, except for the "INPUT" key.',
+				'Whether to purge the default request queue, dataset and key-value store before the run starts.',
 			required: false,
+			default: true,
+			allowNo: true,
+			parse: async (input, context) => {
+				// Resurrect is a special case, akin to --no-purge
+				if (context.argv.some((arg) => arg === '--resurrect')) {
+					usedResurrectFlag = true;
+					return false;
+				}
+
+				return input;
+			},
+			aliases: ['resurrect'],
```

Comment:
is this correct? `resurrect` is an alias for `no-purge`, not for `purge`

i wonder if it would be better to have both flags declared property and disallow mixing them

Reply:
We can either define it standalone or keep it as is, your choice

Reply:
In fact, if we want it to show up in --help, we need to define it standalone

Reply:
i am just concerned if this wont look like `--resurrect` is alias for `--purge`, which is not true

Reply:
Well it doesn't even show up rn, so I'll have to define it separately instead

![Code - 2025-01-16 at 11 40 57](https://github.com/user-attachments/assets/2b7e9a1f-bee1-4a68-bf06-724e5605b9ac)

Reply:
Mind the stacktrace, it's because its running in dev mode 
![Code - 2025-01-16 at 11 48 58](https://github.com/user-attachments/assets/884a08db-8f55-4db5-9d83-8cc2bfd6b3af)

Reply:
i would also mention in the `purge` description that its the default for crawlee projects. 

also i just verified with others that the autopurging works the same in crawlee for python, not sure if the PR is also respecting that?

Reply:
![Code - 2025-01-16 at 11 58 33](https://github.com/user-attachments/assets/dcd918c4-a97d-4130-9831-b329624a4af9)

Good?